projects/components/src/lib/infinite-scroll/infinite-scroll.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="infinite-scroll-container overflow-auto" [style.height]="containerHeight" (scroll)="onContainerScroll($event)">
+  <div>
+    <div *ngFor="let item of items" class="item-scroll" [class.hover:bg-blue-100]="selectedItem !== item.label" [class.bg-blue-100]="selectedItem === item.label" (click)="onItemClick(item)">
```

Comment:
These styles would need to be pulled into _infinite-scroll.scss as these won't be picked up by tailwind when we use it as a component.

Reply:
done